src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+export type Status = 'all' | 'active' | 'completed';
+
+type Props = {
+  todos: Todo[];
+  status: Status;
+  setStatus: (status: Status) => void;
+};
+
+export const ToDoFooter: React.FC<Props> = ({ todos, setStatus, status }) => {
+  const completedTodos = todos.filter(todo => todo.completed);
+  const activeTodos = todos.filter(todo => !todo.completed);
+
+  return (
+    <footer className="todoapp__footer">
+      <span
+        className="todo-count"
+      >
+        {`${activeTodos.length} items left`}
```

Comment:
In the demo, if you filter by completed todos, the count of `todos left` goes to zero and I'm not sure why, the code looks legit. Maybe it needs a re-run deploy?